go-controller/pkg/crd/egressfirewall/v1/types.go

```diff
@@ -0,0 +1,80 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EgressNetworkFirewallRuleType indicates whether an EgressNetworkFirewallRule allows or denies traffic
+// +kubebuilder:validation:Pattern=^Allow|Deny$
+type EgressFirewallRuleType string
+
+const (
+	EgressFirewallRuleAllow EgressFirewallRuleType = "Allow"
+	EgressFirewallRuleDeny  EgressFirewallRuleType = "Deny"
+)
+
+// +genclient
+// +resource:path=egressfirewall
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
Do we want some user facing printer columns for the CRD? It might help the usability of this feature, for example: the status field (in case you planned on changing it and adding some fields to `EgressFirewallStatus`)?

Reply:
+1

Reply:
@abhat @alexanderConstantinescu made a user facing printer column for the status of egressfirewall